test-scripts/entrypoint

```diff
@@ -113,8 +113,14 @@ if [ "x$2" == "xserver" -a $# -ge 6 ]; then
             echo PBS_SERVER=$(hostname -s)
             _svr=$(hostname -s)
         else
+	    if [ $3 -eq 1 ]; then
```

Comment:
I don't think you need this if-else block. You are setting _svr to "pbs-server-$3" in both cases, and you are also echo'ing PBS_SERVER=pbs-server-1 in both cases (as in the if block, $3 == 1). The only difference is echo'ing PBS_SERVER_HOST_NAME, which also you could do in both cases, there's no harm setting it/echo'ing it in either case right?

In short, I think this can be replaced with:
echo PBS_SERVER=pbs-server-1
echo PBS_SERVER_HOST_NAME=pbs-server-$3
_svr="pbs-server-$3"